# Objectives of the BRB-seq transcriptomic analysis pipeline
This notebook reproduces the main steps of the BRB-seq transcriptomic analysis used in the article entitled <ins> Unveiling the role of mir-187 in adult ovarian follicle growth and female fecundity in medaka (Oryzias latipes). </ins>

It focuses on two key objectives:

**- Differential Gene Expression and KEGG Pathway Analysis**
To compare transcriptomic profiles between miR-187⁻/⁻ mutants and wild-type (WT) medaka ovaries, and to explore the biological pathways affected by miR-187 deletion.

**- Identification of Potential miR-187-3p Targets** 
To identify and characterize genes potentially regulated by miR-187-3p that may contribute to ovarian function.

---

# Overview of data processing to be done upstream of the BRB-seq transcriptomic analysis pipeline.
Raw read alignment and gene quantification were carried out using the [Nextflow RNAseq pipeline](https://forgemia.inra.fr/lpgp/rnaseq), developed and maintained by the INRAE LPGP bioinformatics team.  

This automated workflow includes:  
Transcriptome alignment and quantification with *Alevin-fry* ([PMID: 37034702](https://pubmed.ncbi.nlm.nih.gov/37034702/)) via its simpleaf wrapper,  
Genome alignment and exon-level quantification with *STARsolo* ([PMID: 23104886](https://pubmed.ncbi.nlm.nih.gov/23104886/)).  
Execution was performed on an HPC cluster using SLURM job scheduling, following the pipeline configuration.  

The resulting gene-level count matrices (STAR_SOLO_counts.xls) served as input for the downstream analyses described below.

---
# File structure
The following directories indicate where to place input files and where the output generated by the scripts will be stored:  
**Input directory**: `data/input/`  
> Contains the source files required to run the analyses  
> *(e.g., STAR_SOLO_counts, orthologs.txt.gz, miR187_contextscore_output_Camille)*

**Output directory**: `results/output/`  
> Contains the files generated by the scripts  
> *(e.g., tables and plots)*




